docker/fullnode-x/README.md

```diff
@@ -0,0 +1,18 @@
+
+0. Build the containers locally to your desktop/laptop with:
```

Comment:
Why does this start with 0?

Reply:
It's the "before others" step. :-) but really because I'm a SWE and index by zero properly.